### ISSUE TITLE
Sport flashing manual should mention firmware variants

### ISSUE THREAD

**Issue by schwabe** [opentx/opentx-taranis-manual#43]
It would be nice if the s.port flashing manual had  a few short sentences description the three different firmware (FCC, EU-LBT and now outdated old EU) and tell users that these need to match all rx and tx modules.

**Comment by RCdiy** [comment]
Yes... At one point I wanted to include something similar in regards to EU build options but was advised against it but I agree with you. The reason against was that it was beyond the scope of OpenTX.

I recently learnt (from EEVblog I think) that the "FCC Declaration of Conformity" is not actually tested/verified by the FCC. It is just a declaration. Many reputable manufacturers post the conformity certificate on their websites which are issued by accredited test firms. 

https://en.wikipedia.org/wiki/FCC_Declaration_of_Conformity

UL or CUL Underwriters Laboratories is actually tested by UL before that logo can legally go on. 

Point being one is a decleration the other a test/verification...

**Comment by RCdiy** [comment]
Some wording with the build options as well...
